### PR TITLE
Startup Project Idea

### PR DESCRIPTION
**Pull request by gidglass**
Added project idea for the class.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +106,12 @@ Read about guest lectures [here](course/guest-lectures.md).
 * [Naveen Parthasarathy](people/naveen-parthasarathy.md)
 * [Nicolas Joseph](people/nicolas-joseph.md)
 * [Nwamaka Nzeocha](people/nwamaka-nzeocha.md)
+<<<<<<< HEAD
```
Comment (dblock) [inline README.md:109]: There's an unresolved conflict here.

**Comment by dblock** [comment]
Can you please resolve the conflict above, and _squash_ your commits. This might be helpful: http://code.dblock.org/2015/08/31/getting-out-of-your-first-git-mess.html

**Comment by karnseth** [comment]
Can be closed

**Comment by karnseth** [comment]
Hey Gideon,

Let me know if you need help with this (email/slack)
- Karn

**Pull request closed without merging**